marlowe-cli/cli-test/Language/Marlowe/CLI/Test/CLI/Types.hs

```diff
@@ -135,17 +151,34 @@ instance FromJSON MarloweValidators where
     inTx <|> fromRuntimeRegistry <|> fromPublished
 
 instance ToJSON MarloweValidators where
-  toJSON InTxCurrentValidators = Aeson.String "inTxCurrent"
-  toJSON ReferenceRuntimeValidators = Aeson.String "referenceRuntime"
+  toJSON InTxCurrentValidators = A.String "inTxCurrent"
+  toJSON ReferenceRuntimeValidators = A.String "referenceRuntime"
   toJSON (ReferenceCurrentValidators permanent publisher) =
-    Aeson.object
+    A.object
       [ "referenceCurrent"
-          .= Aeson.object
-            [ "permanent" .= permanent
-            , "publisher" .= publisher
-            ]
+          .= A.object ["permanent" .= permanent, "publisher" .= publisher]
       ]
 
+data AnalysisStrategy = AnalysisStrategy
```

Comment:
Nice!

Reply:
Brian, regarding cardano node agnostic analytical tools (based on ScriptContext etc.) integartion - I'm still missing the full integration, I have pieces in place. I used it for simple testing but this would postpone the final PR even more.
Maybe I should comment out the types and add a ticket to the backlog because it can be confusing.
I'm using full tx buildup (in the simulated env) to calculate ExUnits in this PR only: https://github.com/input-output-hk/marlowe-cardano/blob/6b89cc634a9739e5ffb508747471c58e59e54097/marlowe-cli/src/Language/Marlowe/CLI/IO.hs#L508